docs/edgeware-runtime/parameters.md

```diff
@@ -16,7 +16,7 @@ This page reflects the runtime [file of the chain viewable at Github](https://gi
 | **Parameter** | Value | Description |
 | :--- | :--- | :--- |
 | **Reaping Threshold** | 0.001 EDG | The minimum EDG required in the account balance to create or maintain an account. |
-| **Transaction Minimum** | ??? | The Min. amount you can send to an Edgeware Address. |
+| **Transaction Minimum** | 1e-18 | The Min. amount you can send to an Edgeware Address. |
```

Comment:
This isn’t right. It isn’t the lowest unit possible because the fee is likely larger.

Reply:
Yes but in case of the batched transaction it is possible to transfer such low amount to multiple addresses, no? If this meant for indicating lowest unit then should we simply replace 'Transaction Minimum' with 'Decimals' (18)? @drewstone